vcd/resource_vcd_independent_disk_test.go

```diff
@@ -69,8 +99,8 @@ func testAccCheckDiskCreated(itemName string) resource.TestCheckFunc {
 			return fmt.Errorf(errorRetrievingVdcFromOrg, testConfig.VCD.Vdc, testConfig.VCD.Org, err)
 		}
 
-		disk, _ := vdc.QueryDisk(name)
-		if disk == (govcd.DiskRecord{}) {
+		_, err = vdc.GetDisksByName(diskName, true)
```

Comment:
If we search by name, we risk getting duplicated ones.

Reply:
updated